apps/src/templates/BaseDialog.jsx

```diff
@@ -81,7 +81,7 @@ var BaseDialog = React.createClass({
     }
     if (this.props.fullHeight) {
```

Comment:
This condition was added recently for a feature that got re-vamped to not use it. No other use cases in the code, so adjusting this value should have no ill effect.

Reply:
If it's not used should we follow up by removing the code?

Reply:
Sorry, I misunderstood this. This sounds good.